plugins/arDominionB5Plugin/modules/settings/templates/diacriticsSuccess.php

```diff
@@ -0,0 +1,57 @@
+<?php decorate_with('layout_2col.php'); ?>
+
+<?php slot('sidebar'); ?>
+
+  <?php echo get_component('settings', 'menu'); ?>
+
+<?php end_slot(); ?>
+
+<?php slot('title'); ?>
+    <h1><?php echo __('Diacritics settings'); ?></h1>
+<?php end_slot(); ?>
+
+<?php slot('content'); ?>
+
+  <div class="alert alert-info">
+    <p><?php echo __('Please rebuild the search index after uploading diacritics mappings.'); ?></p>
+    <pre>$ php symfony search:populate</pre>
+  </div>
+
+  <?php echo $form->renderGlobalErrors(); ?>
+
+  <?php echo $form->renderFormTag(url_for(['module' => 'settings', 'action' => 'diacritics']), ['method' => 'post']); ?>
+
+    <?php echo $form->renderHiddenFields(); ?>
+
+    <div id="content">
+
+      <table class="table sticky-enabled">
+        <thead>
+          <tr>
+            <th width="30%"><?php echo __('Name'); ?></th>
+            <th><?php echo __('Value'); ?></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td><?php echo __('Diacritics'); ?></td>
+            <td><?php echo $form->diacritics; ?></td>
+          </tr>
+          <tr>
+            <td><?php echo __('Mappings YAML'); ?></td>
+            <td>
+              <span><?php echo __('YAML file containing char_filter mappings.'); ?></span></br>
+              <?php echo $form->mappings; ?><br/>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
```

Comment:
We are not using the table display on any B5 settings page anymore. Like for B2, you could follow the template from the CSV Validator settings.

---

For the file input, there is a way to style them better on B5, check any of the import pages or the finding aid upload form:

https://github.com/artefactual/atom/blob/qa/2.x/plugins/arDominionB5Plugin/modules/informationobject/templates/uploadFindingAidSuccess.php#L38

I'm not sure if that would be enough to display the validation errors, but give a try uploading a not accepted media type.

Reply:
hi @jraddaoui , i've pushed some changes! thank you :)